README.md

```diff
@@ -15,6 +15,29 @@ Docs: [![Documentation Status](https://inch-ci.org/github/samvera/sufia.svg?bran
 Jump in: [![Slack Status](http://slack.samvera.org/badge.svg)](http://slack.samvera.org/)
 [![Ready Tickets](https://badge.waffle.io/samvera/sufia.png?label=ready&title=Ready)](https://waffle.io/samvera/sufia)
 
+
+# Shift in Community Focus
+
+During 2017, the [Samvera community](http://samvera.org/) consolidated Sufia and [CurationConcerns](https://github.com/samvera/curation_concerns) into [Hyrax](https://github.com/samvera/hyrax):
+
+* [planning notes on consolidation](https://wiki.duraspace.org/pages/viewpage.action?pageId=78161232))
```

Comment:
This is on our old wiki. Can you link to https://wiki.duraspace.org/pages/viewpage.action?pageId=87461232 instead?